org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/_support/AbstractComponentInstanceResource.java

```diff
@@ -506,7 +508,12 @@ public Response updateDefinitions(InputStream requestBodyStream) {
             return Response.ok().entity(validationError).build();
         }
     }
-
+    
+    @Path("edmm")
```

Comment:
Why is this in the abstract class? Shouldn't this be added to the `ServiceTemplateInstanceResource`?

Reply:
Do you mean `ServiceTemplateResource.java` ?

Reply:
If everything related to a service template instance is handled in `ServiceTemplateResource.java`, yes :-)